src/components/Toolbar.tsx

```diff
@@ -51,6 +54,21 @@ const Toolbar: FC<ToolbarProps> = ({ customize, onSelect, selected }) => {
    * Methods
    **********************************************************************/
 
+  /** Deselects the toolbar button. */
+  const deselectPressingToolbarId = useCallback(() => {
+    const delay = Math.max(0, ANIMATION_MS - (Date.now() - latestPress))
```

Comment:
A brief comment here to explain how this math/logic works would be helpful.

Reply:
Good point. This statement/calculation is a bit complex for one uncommented line.

In cases like these, I think it's even nicer to break up the math into multiple statements and use well-named intermediate variables. I've added `timeSinceLastPress` and `timeUntilAnimationEnd` to help clarify the calculations and added a comment explaining the usage of `Math.max`: c1f6b8a3b60e8295c41240eb6b7c77855dcc730c